docs/getting-started/release-notes/v2_9_0.md

```diff
@@ -30,10 +50,45 @@ Zowe Version 2.9.0 contains the bug fixes that are described in the following to
 
 ### Zowe API Mediation Layer
 
+### Imperative CLI Framework
+
+- Updated handle logic to display a warning in cases where a `null` command handler is provided.
```

Comment:
Maybe should be "Added logic to display a warning in cases where a `null` command handler is provided."